pull_request_template.md

```diff
@@ -1,39 +1,40 @@
 ## Overview
+<!-- Please provide a brief summary of the changes proposed in this pull request. Make sure branch name matches the ticket below. -->
 
-Please provide a brief summary of the changes proposed in this pull request. Make sure branch name matches the ticket below.
 
 ## Ticket
+<!-- [NEX-XXXX](URL) -->
 
-[NEX-XXXX](URL)
```

Comment:
Add something like 
<!--- Please include the link to the Jira Ticket here -->

Reply:
![jira copy](https://user-images.githubusercontent.com/124634523/219767687-7454771c-8149-4624-9743-3ec4f43bf51e.PNG)
If you copy that link (copy, not copy link address) and paste it in github it will be automatically formatted as a markdown link

Reply:
Ohhh nice! Could you add that in the comment as well so others do that method of copying

Reply:
Done